pkg/controller/workload/job/job_controller_test.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package job
+
+import (
+	"testing"
+
+	batchv1 "k8s.io/api/batch/v1"
+
+	"sigs.k8s.io/kueue/pkg/util/pointer"
```

Comment:
one block

Reply:
Actually, verification started to fail once I joined. It seems the newline separates the k8s and kueue packages. An analogous line is present in other go files.

Reply:
ah, yes, this is a kueue package.